SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosTaskBuilder.java

```diff
@@ -381,6 +381,10 @@ private void prepareCustomExecutor(final TaskInfo.Builder bldr, final Singularit
   private void prepareCommand(final TaskInfo.Builder bldr, final SingularityTaskId taskId, final SingularityTaskRequest task, final Protos.Offer offer, final Optional<long[]> ports) {
     CommandInfo.Builder commandBldr = CommandInfo.newBuilder();
 
+    if (task.getDeploy().getUser().isPresent()) {
+      commandBldr.setUser(task.getDeploy().getUser().get());
```

Comment:
Can we add this corresponding change to `prepareCustomExecutor()` too? I can see some utility in being able to control what user the custom executor is launched as.